src/MagnumPlugins/BasisImporter/Test/BasisImporterTest.cpp

```diff
@@ -190,55 +223,51 @@ void BasisImporterTest::importTwice() {
     CORRADE_VERIFY(importer);
     CORRADE_COMPARE(importer->configuration().value<std::string>("format"),
-        formatData.expectedFormatStr);
+        formatData.suffix);
     CORRADE_VERIFY(importer->openFile(Utility::Directory::join(BASISIMPORTER_TEST_DIR,
-        formatData.needsPow2 ? "rgb_2_images_pow2.basis" : "rgb_2_images.basis")));
+        formatData.file)));
 
     /* Verify that everything is working the same way on second use */
     Containers::Optional<Trade::ImageData2D> image = importer->image2D(0);
     CORRADE_VERIFY(image);
     CORRADE_COMPARE(image->compressedFormat(), formatData.expectedFormat);
-    if(formatData.needsPow2) CORRADE_COMPARE(image->size(), (Vector2i{64, 32}));
-    else CORRADE_COMPARE(image->size(), (Vector2i{63, 27}));
+    CORRADE_COMPARE(image->size(), formatData.expectedSize);
 
     /* Verify that the 90° rotated second image can be loaded also */
     image = importer->image2D(1);
     CORRADE_VERIFY(image);
     CORRADE_COMPARE(image->compressedFormat(), formatData.expectedFormat);
-    if(formatData.needsPow2) CORRADE_COMPARE(image->size(), (Vector2i{32, 64}));
-    else CORRADE_COMPARE(image->size(), (Vector2i{27, 63}));
+    CORRADE_COMPARE(image->size(), (Vector2i{formatData.expectedSize.y(), formatData.expectedSize.x()}));
 }
 
 void BasisImporterTest::rgba() {
     auto& formatData = FormatData[testCaseInstanceId()];
     const std::string pluginName = "BasisImporter" + std::string(formatData.suffix);
-    setTestCaseDescription(pluginName);
+    setTestCaseDescription(formatData.suffix);
 
     Containers::Pointer<AbstractImporter> importer = _manager.instantiate(pluginName);
     CORRADE_VERIFY(importer);
     CORRADE_COMPARE(importer->configuration().value<std::string>("format"),
-        formatData.expectedFormatStr);
+        formatData.suffix);
     CORRADE_VERIFY(importer->openFile(Utility::Directory::join(BASISIMPORTER_TEST_DIR,
-        formatData.needsPow2 ? "rgba_2_images_pow2.basis" : "rgba_2_images.basis")));
+        formatData.fileAlpha)));
 
     /* Verify that everything is working the same way on second use */
     Containers::Optional<Trade::ImageData2D> image = importer->image2D(0);
     CORRADE_VERIFY(image);
     CORRADE_COMPARE(image->compressedFormat(), formatData.expectedFormatAlpha);
-    if(formatData.needsPow2) CORRADE_COMPARE(image->size(), (Vector2i{64, 32}));
-    else CORRADE_COMPARE(image->size(), (Vector2i{63, 27}));
+    CORRADE_COMPARE(image->size(), formatData.expectedSize);
 
     /* Verify that the 90° rotated second image can be loaded also */
     image = importer->image2D(1);
     CORRADE_VERIFY(image);
     CORRADE_COMPARE(image->compressedFormat(), formatData.expectedFormatAlpha);
-    if(formatData.needsPow2) CORRADE_COMPARE(image->size(), (Vector2i{32, 64}));
-    else CORRADE_COMPARE(image->size(), (Vector2i{27, 63}));
+    CORRADE_COMPARE(image->size(), (Vector2i{formatData.expectedSize.y(), formatData.expectedSize.x()}));
```

Comment:
`.flipped()` ;)

Reply:
Even checked the docs because I guessed there would be something like this :D

Reply:
There *always* is :P